.github/ISSUE_TEMPLATE/apm-server-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: APM Server issue template
+about: General Issue template
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Hello!
+
+We like to keep the Github repository for product issues - verified bug reports, pull requests, apm-server specific feature requests, etc and have users ask questions about how to use the product or a particular feature in our discuss forums.
+
+Head on over to our official [Discuss forum](https://discuss.elastic.co/c/apm) and let us know there what's on your mind. You will reach a wider audience, get help from the broader community, and maybe learn how others solved problems similar to yours.
+
+
+Thanks!
```

Comment:
This statement now sounds to me like we are almost assuming that users probably hit this while they should really open a discuss post. I wouldn't feel encouraged to open an issue when reading this. 

How about after the first paragraph we add something along the lines of:
_Therefore you are very welcome to open an issue for above mentioned reasons. Otherwise, head on over..._
(Definitely not the perfect wording in my example, but it should do for sketching the idea).

Reply:
is https://github.com/elastic/apm-server/pull/2668/files#r322282672 any better?

Reply:
Yes, I'd say it is better. Thanks for changing.